test/run

```diff
@@ -1,13 +1,13 @@
 #!/bin/sh
 case `pwd` in
 	*/test/unittests)
-		PYTHONPATH=../.. nosetests -w . "$@"
+		PYTHONPATH=../.. nosetests -v -w . "$@"
```

Comment:
No I notices that we are still using `nose`... And it's deprecated.
We should move to `pytest` in the future. What do you think? It should be super easy

Reply:
OK, I will remove this commit from this PR, and create a new one to master branch to use pytest, and then check whether we could backport to crmsh-3.0 branch